en/admin/admin_utils.rst

```diff
@@ -819,7 +819,7 @@ The following options can be used in client/server mode only.
 
 .. option:: -i, --input-class-file=FILE
 
-    You can specify an input file name that contains the table name with this option. Write one table name in a single line; invalid table name is ignored. Note that you cannot specify the list of the table names after a database name in case of you use this option.
+    You can specify an input file name that contains the table name with this option. Write one table name in a single line; invalid table name is ignored. Note that you cannot specify the list of the table names after a database name in case of you use this option. If you use this option on client/server mode, It is skipped freeing objects such as catalog, delete files and tracker etc.
```

Comment:
It is skipped freeing objects such as catalog, delete files and tracker etc.
-->
freeing (cleanup of?) objects such as catalog, delete files and tracker, etc. is not performed.
or 
it skips freeing (cleanup of?) objects such as catalog, delete files and tracker, etc.

Reply:
I modified it as before.